omnisharp.el

```diff
@@ -456,7 +457,7 @@ cursor at that location"
 (defun omnisharp--eldoc-worker ()
   "Gets type information from omnisharp server about the symbol at point"
   (omnisharp--completion-result-get-item 
-   (omnisharp-post-message-curl-as-json
+   (omnisharp-post-http-message
```

Comment:
Looks like this was previously unsupported. Does it work for you now?
It will need some fixing to support stdio, but let's leave it like you have done for now.